src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -8,10 +8,10 @@
 // distribute, sublicense, and/or sell copies of the Software, and to
 // permit persons to whom the Software is furnished to do so, subject to
 // the following conditions:
-// 
+//
```

Comment:
Ack I'm sorry! :-( You can turn off "trim on save" in both extensions so that you can still see whitespace but it won't touch it.